### PR TITLE
SG-29856 Replace shotgridsoftware references

### PR DESCRIPTION
**Pull request by carlos-villavicencio-adsk**

### CODE REVIEW

**Comment by ghost** [comment]
[Documentation Preview](http://sg-devdocs.ems.autodesk.com/tk-doc-generator/ce08ef9bd37aa1d697defbb364550b1cfbfe77ce/index.html)

File: Dockerfile [hunk 1]
```diff
@@ -8,7 +8,7 @@
 # dockerfile to spin up a container suitable for generation of documentation
 
 FROM centos:7
-LABEL maintainer="toolkit@shotgunsoftware.com"
+LABEL maintainer="https://shotgridsoftware.com/"
```
Comment (julien-lang) [inline Dockerfile:11]: Is this information we can see on the generated doc? Does that work?

Comment (carlos-villavicencio-adsk) [reply Dockerfile:11]: This only adds metadata to the Docker image according to [official docs](https://docs.docker.com/engine/reference/builder/#label).

**Review by julien-lang — APPROVED** [review]
LGTM!

**Pull request merged**